src/main/java/com/esri/samples/map/create_and_save_map/CreateAndSaveMapController.java

```diff
@@ -147,12 +150,11 @@ void authenticate() {
           } else if (portal.getLoadStatus() == LoadStatus.FAILED_TO_LOAD) {
 
             // show alert message on error
-            showMessage("Authentication failed", portal.getLoadError().getMessage(), Alert.AlertType.ERROR);
+            new Alert(Alert.AlertType.ERROR, "Authentication failed: " + portal.getLoadError().getMessage()).show();
           }
         });
 
-        // loading the portal info of a secured resource
-        // this will invoke the authentication challenge
+        // load the portal info of a secured resource. This will invoke the authentication challenge
```

Comment:
just spotted a typo from the original code, but GitHub won't allow me to add comments to existing code :D can you correct the spelling of "unsuccessful" on line 205 please?